library/src/compute/construct.rs

```diff
@@ -45,6 +45,7 @@ cast_from_value! {
 ///
 /// - Booleans are converted to `0.0` or `1.0`.
 /// - Integers are converted to the closest 64-bit float.
+/// - Ratios are divided by 100%.
```

Comment:
`100%` equals to `1`, I think that you mean that their values are divided by `100` ?

Reply:
I think they meant that `float(40%) == 40% / 100%` as the latter division is already supported by Typst.